wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/impl/SourceStorageParametersValidator.java

```diff
@@ -41,6 +41,16 @@ public void validate(SourceStorage source, FactoryParameter.Version version)
                     entry.getValue()));
           }
           break;
+        case "skipFirstLevel":
+          final String skipFirstLevel = entry.getValue();
+          if (!"true".equals(skipFirstLevel) && !"false".equals(skipFirstLevel)) {
+            throw new ConflictException(
+                format(
+                    PARAMETRIZED_ILLEGAL_PARAMETER_VALUE_MESSAGE,
+                    "source.project.parameters.skipFirstLevel",
```

Comment:
As far as I see there is not `project` field in `source` object while `project` object has `source` field. Should not it be `"project.source.parameters.skipFirstLevel"`?

Reply:
ty, fixed